ppr-ui/src/components/tombstone/TombstoneDefault.vue

```diff
@@ -3,9 +3,28 @@
     <v-col v-if="isStaff" class="staff-header" cols="1"></v-col>
     <v-col :cols="isStaff? '11' : '12'" :class="isStaff? 'pl-4' : ''">
       <div class="ma-0 pa-0">
-        <span class="tombstone-header">
-          <b>{{ header }}</b>
-        </span>
+        <v-row no-gutters class="justify-space-between align-baseline">
+          <span class="tombstone-header">
+            <b>{{ header }}</b>
+          </span>
+          <v-tooltip top content-class="top-tooltip pa-5" nudge-left="30">
+            <template  v-slot:activator="{ on, attrs }">
+              <a :href="'https://www2.gov.bc.ca/gov/content/employment-business/business/managing-a-business/'
+              +'permits-licences/news-updates/modernization-updates/modernization-resources#ppr'"
+              class="text-decoration-none" target="_blank" rel="noopener noreferrer">
+                <div v-bind="attrs" v-on="on">
+                  <v-row no-gutters class="align-center">
+                    <v-icon left color="primary">mdi-help-circle-outline</v-icon>
+                    <span class="primary--text" style="font-size: 16px">Help</span>
```

Comment:
Is the `style` font-size required here? 
What is the default, without this?

Reply:
no it's not necessary, the default is 16px, I included this because the spec explicitly pointed 16px.
I'll remove it.